docs/Learning-Environment-Examples.md

```diff
@@ -273,7 +281,15 @@ If you would like to contribute environments, please see our
   * Vector Action space: (Continuous) 3 corresponding to agent force applied for
     the jump.
   * Visual Observations: None.
-* Reset Parameters: None.
+* Reset Parameters: Two, corresponding to the following
+    * Agent_scale:
+        * Default: 1
+        * Recommended Minimum: 0.25
+        * Recommended Maximum: 5
```

Comment:
Had issues with Bouncer at 4-5... agent doesn't bounce

Reply:
Changed reset parameter from agent_scale to including banana movement in the vertical axis.

Reply:
Hey @sankalp04, I just thought of an issue with using banana movement - since it's on-demand, the agent only gets the location of the banana when it hits the ground. So it won't see where the banana is once it jumps, if it's moving. Maybe we can make it banana height, or only move before CollectObservations()?